styles/index.css

```diff
@@ -82,10 +94,23 @@ form {
   border-radius: 5px;
   padding: 10px;
   width: 95%;
+  background-color: black;
+  color: white;
+}
+
+.form-control button {
+  border: 2px solid yellowgreen;
+  border-radius: 5px;
+  padding: 10px;
+  width: 95%;
+  background-color: black;
+  color: white;
```

Comment:
Great!
try adding `cursor: pointer` to show the mouse cursor when you hover over the button. Also, you can add box shadows to the button, i.e

```
.form-control button:hover {
// add box shadows styles 
}
```